packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -34,7 +34,9 @@ export default {
       // hold the last 2 minute of a calls data
       historyMax: 120,
       // Once packet loss hits this ratio, the video will downgrade
-      videoPacketLossRatioThreshold: 9
+      videoPacketLossRatioThreshold: 9,
+      rttThreshold: 500, // 500 ms
+      jitterThreshold: 500 // 500 ms
```

Comment:
this may need to change between audio/video?  Not sure - also Jitter threshold may need to altered